docs/tutorial/visualization/open3d_visualizer_advanced.rst

```diff
@@ -0,0 +1,511 @@
+
+
+.. _rendering_models:
+
+Rendering Models
+::::::::::::::::
+
+Rendering ``TriangleMesh``'es of 3D Models
+""""""""""""""""""""""""""""""""""""""""""
+
+In the Basic Open3D Visualizer Tutorial, we demonstrated how to apply materials manually to built-in Open3D geometries. It is also possible to load *TriangleMesh*'es from full 3D models using the ``o3d.io.read_triangle_mesh()`` method, as shown below:
+
+
+.. code-block:: python
+
+  >>> monkey_model = o3d.io.read_triangle_mesh('examples/test_data/monkey/monkey.obj')
+  >>> vis.draw(monkey_model)
+
+That will automatically apply the default material which exists in a 3D model:
+
+.. image:: https://user-images.githubusercontent.com/93158890/160008560-4834c962-efa7-4d69-b99d-9ff321a03c02.jpg
+    :width: 700px
+
+
+Next, we will learn how to render full 3D models in all their glory.
+
+
+
+Rendering Full 3D Models
+""""""""""""""""""""""""
+
+In the Basic Open3D Visualizer Tutorial, we rendered *TriangleMesh* and *Tensor-based TriangleMesh* objects. But the ``draw()`` function can also render full-fledged 3D models containing a set of textures and material properties. To read a complete model, we need to use the ``open3d.io.read_triangle_model()`` method, which imports all the material properties in addition to the *TriangleMesh*:
+
+.. code-block:: python
+
+    >>> monkey_model = o3d.io.read_triangle_model('examples/test_data/monkey/monkey.obj')
+    >>> vis.draw(monkey_model)
+
+Clearly, a staggering difference in rendering:
+
+.. image:: https://user-images.githubusercontent.com/93158890/148611141-d424fc74-be7e-4833-913c-714fc3c4fbd2.jpg
+    :width: 700px
+
+
+
+Rendering More Complex Models
+:::::::::::::::::::::::::::::
+
+In the previous section (:ref:`rendering_models`) we have covered how to render complete 3D models with the ``open3d.io.read_triangle_model()`` method. This method can also handle more complex models containing a collection of materials and parts (sub-models) from which the complete object gets assembled.
+
+For this example, we will need to download / ``clone`` *glTF-Sample-Models*  from the KhronosGroup. `glTF (GL Transmission Format) <https://docs.fileformat.com/3d/gltf/>`_ is a 3D file format that stores 3D model information in JSON format. 
+
+First, **minimize your current Python terminal session and open a new one. In a new terminal session:**
+
+.. image:: https://user-images.githubusercontent.com/93158890/150047410-de591582-67c5-42bd-b644-764c36b8c4b8.jpg
+    :width: 800px
+
+1. Change Directory (``cd``) to where you would like the *glTF-Sample-Models* repository to be copied;
+2. Use the ``git clone`` command to download the *glTF-Sample-Models* repository:
+
+.. code-block:: sh
+
+    $ git clone https://github.com/KhronosGroup/glTF-Sample-Models
+
+3. Wait for the cloning process to complete. The command prompt will return when the process is done.
+4. Close the command prompt window you've just used for the ``git clone`` command.
```

Comment:
Simplify this section to:

```rst
From a command prompt,  download sample glTF models from the Khronos Group's
 git repository with:

.. code-block:: sh
    $ git -C .. clone https://github.com/KhronosGroup/glTF-Sample-Models
```

Reply:
Rishabh is working on integrating this model into a dataset. As soon as he is done, we will greatly simplify this section.

Reply:
Resolved with Rishabh's datasets.